### PR TITLE
[LIBCLOUD-892] Improve GCE Compute driver docstrings.

### PR DESCRIPTION
**Pull request by Mac21**
### Description

Replace this with the PR description (mention the changes you have made, why
you have made them, provide some background and any references to the provider
documentation if needed, etc.).

For more information on contributing, please see [Contributing](http://libcloud.readthedocs.org/en/latest/development.html#contributing)
section of our documentation.

### Status

Replace this: describe the PR status. Examples:

- work in progress
- done, ready for review

### Checklist (tick everything that applies)

- [ ] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [ ] Documentation
- [ ] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by Mac21** [comment]
**Fixed doc strings that were copied, but not updated to the correct parameter.**

**Description**

I fixed the doc strings of the following functions:

[ex_copy_image](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_copy_image)

1. the url param annotation was being used as the type annotation

[ex_create_image](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_create_image)

1. the keyword annotation was typo'd to keywork.

[ex_delete_access_config](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_delete_access_config)

1. the name keyword's type annotation was copied from node.

[ex_get_image](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_get_image)

1. the ex_project_list parameter's type annotation was using ',' instead of 'or' for multiple types.

[ex_get_license](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_get_license)

1. There were two annotations for name when the project annotation was missing.

[ex_get_subnetwork](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_get_subnetwork)

1. There were two annotations for name when the region annotation was missing.

[ex_instancegroup_list_instances](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_instancegroup_list_instances)

1. The instancegroup type annotation was using double back ticks around the type instead of single ticks.

[ex_destroy_subnetwork](http://libcloud.readthedocs.io/en/stable/apidocs/libcloud.compute.drivers.html#libcloud.compute.drivers.gce.GCENodeDriver.ex_destroy_subnetwork)

1. There were two annotations for name when the region annotation was missing.

**Status**
done, ready for review

File: libcloud/compute/drivers/gce.py [hunk 1]
```diff
@@ -1849,7 +1849,7 @@ def ex_delete_access_config(self, node, name, nic):
         :type     node: ``Node``
 
         :keyword  name: Name of the access config.
-        :type     node: ``str``
+        :type     nam: ``str``
```
Comment (tonybaloney) [inline libcloud/compute/drivers/gce.py]: guess this should be `name`?

**Review by tonybaloney — CHANGES_REQUESTED** [review]
thanks for the contribution, just one small change

**Comment by Mac21** [comment]
Sorry about that! I fixed the typo and found another error that was missed.

**Comment by tonybaloney** [comment]
thanks :+1: merging

**Pull request closed without merging**